libs/server/AOF/AofProcessor.cs

```diff
@@ -31,11 +31,13 @@ public sealed unsafe partial class AofProcessor
         /// Session for main store
         /// </summary>
         readonly ClientSession<SpanByte, SpanByte, SpanByte, SpanByteAndMemory, long, MainStoreFunctions> session = null;
```

Comment:
do we need to keep `session` and `objectStoreSession` here as fields?

Reply:
Changing to add XxxContext.Session; this is primarily used for Dispose() and also to get Session.ID and Session.Version